### PR TITLE
Fixed major SonarQube issues in different components

### PR DESCRIPTION
**Pull request by vbergmann**

### CODE REVIEW

File: src/main/java/org/aludratest/content/edifact/edifatto/EdifattoUtil.java [hunk 1]
```diff
@@ -25,8 +25,9 @@
  * @author Volker Bergmann */
 public final class EdifattoUtil {
 
+    /** Private constructor for preventing instantiation of this utility class. */
     private EdifattoUtil() {
-        // private constructor to prevent instantiation of this utility class
+        // Private constructor for preventing instantiation of this utility class.
```
Comment (falbrech-hsdg) [inline src/main/java/org/aludratest/content/edifact/edifatto/EdifattoUtil.java]: duplicate comment line (identical to method comment). Please remove.

File: src/main/java/org/aludratest/content/xml/util/DatabeneXmlUtil.java [hunk 2]
```diff
@@ -26,6 +26,11 @@
  * @author Volker Bergmann */
 public class DatabeneXmlUtil {
 
+    /** Private constructor for preventing instantiation of this utility class. */
+    private DatabeneXmlUtil() {
+        // Private constructor for preventing instantiation of this utility class.
```
Comment (falbrech-hsdg) [inline src/main/java/org/aludratest/content/xml/util/DatabeneXmlUtil.java]: duplicate comment , identical to method comment. Please remove.

File: src/main/java/org/aludratest/impl/log4testing/TestCaseLogImpl.java [hunk 3]
```diff
@@ -32,9 +32,9 @@
 
     private TestSuiteLogImpl parent;
 
-    private volatile boolean ignored;
+    private volatile boolean ignored; // NOSONAR 'volatile' is use by purpose here
```
Comment (falbrech-hsdg) [inline src/main/java/org/aludratest/impl/log4testing/TestCaseLogImpl.java]: i would prefer "intentionally" :-) Or "on purpose"...

File: src/main/java/org/aludratest/impl/log4testing/TestStepLogImpl.java [hunk 4]
```diff
@@ -35,29 +35,29 @@
...
+    private volatile String errorMessage; // NOSONAR 'volatile' is use by purpose here
 
-    private volatile Throwable error;
+    private volatile Throwable error; // NOSONAR 'volatile' is use by purpose here
 
-    private volatile String comment;
+    private volatile String comment; // NOSONAR 'volatile' is use by purpose here
 
-    private volatile TestStatus status;
+    private volatile TestStatus status; // NOSONAR 'volatile' is use by purpose here
```
Comment (falbrech-hsdg) [inline src/main/java/org/aludratest/impl/log4testing/TestStepLogImpl.java]: see above for all these comments.

File: src/main/java/org/aludratest/junit/AludraTestJUnitSuite.java [hunk 5]
```diff
@@ -77,7 +77,7 @@ public AludraTestJUnitSuite(Class<?> testClass) throws InitializationError {
         String suiteName = System.getProperty(SUITE_SYSPROP);
 
         if (StringUtil.isEmpty(suiteName)) {
-            LOGGER.debug("SuiteName:\"" + suiteName + "\"");
+            LOGGER.debug("SuiteName:\"{}\"", suiteName);
```
Comment (falbrech-hsdg) [inline src/main/java/org/aludratest/junit/AludraTestJUnitSuite.java]: Does this make sense at all? Suite name is either null or an empty string. Why logging it at all here?

File: src/main/java/org/aludratest/content/edifact/edifatto/EdifattoUtil.java (hunk 1 above)
Comment (vbergmann) [inline src/main/java/org/aludratest/content/edifact/edifatto/EdifattoUtil.java]: SonarQube doesn't like empty methods, including constructors

Comment (vbergmann) [inline src/main/java/org/aludratest/content/edifact/edifatto/EdifattoUtil.java]: OK, but doesn't seem to require a comment in utility class constructors

File: src/main/java/org/aludratest/content/xml/util/DatabeneXmlUtil.java (hunk 2 above)
Comment (vbergmann) [inline src/main/java/org/aludratest/content/xml/util/DatabeneXmlUtil.java]: See above

**Pull request merged**